src/multi_site_inputs_parser.py

```diff
@@ -130,6 +130,11 @@ def multi_site_csv_parser(path_to_csv, api_url, API_KEY, n_sites=None):
 
         path_to_load_files = "./load_profiles"
         add_load_profile_inputs(site_inputs, posts[-1], path_to_load_files=path_to_load_files)
+        
+        path_to_rate_files = "./electric_rates"
+        for x in df['rate_file'].tolist():
+            rate_x = json.load(open(os.path.join(path_to_rate_files, x), "r"))
```

Comment:
Since the `rate_file`s must be JSON, and we put them in the `urdb_response`, maybe we can call `rate_file` something more descriptive? Something like `urdb_json_file`.